cylc/flow/scheduler.py

```diff
@@ -897,8 +903,12 @@ def process_command_queue(self) -> None:
             sep = ', ' if kwargs_string and args_string else ''
             cmdstr = f"{name}({args_string}{sep}{kwargs_string})"
             try:
-                n_warnings: Optional[int] = self.get_command_method(name)(
-                    *args, **kwargs)
+                fcn = self.get_command_method(name)
+                n_warnings: Optional[int]
+                if inspect.iscoroutinefunction(fcn):
+                    n_warnings = await fcn(*args, **kwargs)
```

Comment:
Can we test this line?

Reply:
Yes and are doing!

Without it `async def command_reload_workflow` wouldn't work.